python/lsst/pipe/tasks/maskStreaks.py

```diff
@@ -624,7 +624,7 @@ def _cannyFilter(self, image):
         cannyData : `np.ndarray`
             2-d image of edges found in input image.
         """
-        filterData = image.astype(int)
+        filterData = (image > 0).astype(np.float64)
```

Comment:
We should probably add a comment here explicitly that we are setting pixels to 0 or 1 and making the map a float to be compatible with the canny filter routine.